statistics/dml_update.go

```diff
@@ -142,20 +147,23 @@ func (delta *HistDelta) updateHistDelta(sc *stmtctx.StatementContext, hg *Histog
 	if value == nil {
 		return
 	}
+	// Find the bucket that value falls in.
```

Comment:
Can we not update the histogram in memory ?

Reply:
Do not quite understand it, the following code does not update the histogram.